library/portable-simd/crates/core_simd/src/masks.rs

```diff
@@ -389,6 +390,10 @@ where
     T: MaskElement,
     LaneCount<N>: SupportedLaneCount,
 {
+    /// Converts a SIMD mask to an array of bools.
+    ///
+    /// ## Cost
+    /// Copies the bytes of the array
```

Comment:
```suggestion
    /// Converts a SIMD mask to an array of bools.
```
I will leave the cost section for vary expensive things like heap allocation.